Library/Formula/pacapt.rb

```diff
@@ -0,0 +1,19 @@
+# Pacapt is a wrapper for all major package managers. It allows
+# a user to use only one pacman-likes syntax while working with
+# different package managers.
+
+class Pacapt < Formula
+
+  homepage "https://github.com/letalvoj/pacapt" 
+  url "https://github.com/icy/pacapt.git", :using => :git, :tag => "v2.0.2"
+
+  def install
+    system "make", "install", "BINDIR=#{prefix}"
+    bin.install_symlink "#{prefix}/pacapt" => "pacapt"
```

Comment:
If a use the #{bin} tag I get
install: /usr/local/Cellar/pacapt/2.0.2/bin/pacapt: No such file or directory

like if the bin folder was missing.